misk/src/main/kotlin/misk/security/authz/AccessInterceptor.kt

```diff
@@ -1,5 +1,6 @@
 package misk.security.authz
 
+import jakarta.inject.Inject
```

Comment:
wrong annotation

Reply:
This is just a rearrangement of the imports (my IDE did it automatically)